tests/collections/asr/numba/rnnt_loss/test_rnnt_pytorch.py

```diff
@@ -18,9 +18,13 @@
 import pytest
 import torch
 
-from nemo.collections.asr.losses.rnnt import MultiblankRNNTLossPytorch, RNNTLossPytorch
+from nemo.collections.asr.losses.rnnt import MultiblankRNNTLossPytorch, RNNTLossPytorch, TDTLossPytorch
 from nemo.collections.asr.parts.numba.rnnt_loss.rnnt_numpy import RNNTLoss as RNNTLoss_Numpy
-from nemo.collections.asr.parts.numba.rnnt_loss.rnnt_pytorch import MultiblankRNNTLossNumba, RNNTLossNumba
+from nemo.collections.asr.parts.numba.rnnt_loss.rnnt_pytorch import (
+    MultiblankRNNTLossNumba,
```

Comment:
Please add tests for the new 3 alpha beta gamma kernels + the durations logp kernel in isolation similar to the RNNT alpha beta gamma kernels. If Multiblank doesn't have those tests, please also add them